### DISCUSSION TITLE
OneDNN Execution Provider for INT8 onnx model

### DISCUSSION THREAD

**Discussion by anandvishes** [microsoft/onnxruntime#11309] [Other Q&A]
Hi,
I am working on Inferencing INT8 ONNX model with OneDNN Execution Provider "DnnlExecutionProvider".
The latency is very high when compared to its FP32 model when executed with DnnlEP(almost x100 times).
The same INT8 model works fine with CpuExecutionProvider and OpenvinoExecutionProvider.

The latency for FP32 model was around 5ms but for INT8 model it was 400+ms.
Do we have to handle INT8 model differently for DnnlEP?
Can anyone help on this?

TIA.

Anand.

**Comment by pranavsharma** [comment]
Can you file a github issue instead after filling out the template fully?
